xbmc/input/InputManager.cpp

```diff
@@ -355,6 +355,31 @@ void CInputManager::QueueAction(const CAction& action)
   m_queuedActions.push_back(action);
 }
 
+bool CInputManager::QueueAppCommand(const CAction& action)
```

Comment:
AppCommand is Windows only and should not spoil the rest of the app. Windows needs to send its command the generic way. means AppCommand shold go away completely